.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This is great! Thanks Marcus. I don't see any rules in the `.prettierrc.json` file. Is it stored somewhere else?

Reply:
Also, what do you think of these rules?
```
{
  "semi": false,
  "tabWidth": 2,
  "singleQuote": false,
  "jsxSingleQuote": false,
  "trailingComma": "es5",
  "bracketSpacing": true
}
```

Reply:
I don't really have an opinion with the formatting rules, so i just left it blank for the defaults. Personally, I prefer having semi colons, but I'm cool either way.

Reply:
Ok I added those rules and ran prettier to format the files

Reply:
Okay nice, I am indifferent to the semicolons as well. I'm looking up some best practices and it looks like keeping semi is best

Reply:
Ok I'll change semicolons to true and merge it